### PR TITLE
Added Developers Involved Field for Projects In Progress

### PR DESCRIPTION
**Pull request by arrayslayer**
**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: data/projects.js [hunk 1]
```diff
@@ -96,6 +96,7 @@ projects = [
         ],
         "difficulty" : "high",
         "issues" : [],
+        "developers involved" :[],
```
Comment (yukiisbored) [inline data/projects.js]: I don't think we use spaces for separation, what do you think @hemangsk

Comment (arrayslayer) [inline data/projects.js]: as in the space between developers and involved?

Comment (arrayslayer) [inline data/projects.js]: I looked through the rest of the code. Should I put an _ in between?

Comment (hemangsk) [inline data/projects.js]: Yes adding an underscore will be nice, like for collaborating projects field, we have "collaborating_projects".

**Comment by hemangsk** [comment]
@arrayslayer hey! 😃  Can you please make sure the commit follows the commit guidelines, you have checked the checkboxes but the commit message doesn't look right.  For reference checkout, http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html#quick-reference, and let us know if you have any questions 👍  😃

**Comment by arrayslayer** [comment]
Hey! I added the underscores. I am really sorry for the ill-written commit. I read it up in detail today and applied it. Hopefully, the commit meets the guidelines this time.

**Comment by hemangsk** [comment]
np, its part of the review process, we all are learning. Next step is to actually display this developers involved field to the user,  in the left black sidebar of projects detail popup (that appears when we click on project cards) 👍

**Comment by arrayslayer** [comment]
So, do I have to implement the next step here or is this issue closed?

**Comment by hemangsk** [comment]
This issue only, just do it in a separate commit.

**Comment by meetmangukiya** [comment]
@arrayslayer can you add a screenshot for the same while modifying this field for some project, that'll be really helpful

**Comment by arrayslayer** [comment]
screenshot from 2017-06-25 16-38-48

I added this to implement Developers Involved field for projects. Were you referring to something else?

**Comment by hemangsk** [comment]
@arrayslayer not the screenshot of code but screenshot of the webpage, to view how does it look.

To run this, 
you can do
```
$ git clone https://github.com/coala/projects.git
$ cd projects
$ python -m SimpleHTTPServer 8080
```

**Comment by arrayslayer** [comment]
screenshot from 2017-06-26 09-59-54
This is what comes up.

File: data/projects.js [hunk 2]
```diff
@@ -212,6 +215,7 @@ projects = [
         "tags" : ["Vulture", "AST", "Bears"],
         "markdown": "vulture.md",
         "collaborating_projects" : ["coala", "vulture"]
```
Comment (hemangsk) [inline data/projects.js]: A comma missing after ["coala", "vulture"], therefore no projects are visible on https://github.com/coala/projects/pull/312#issuecomment-310961456

Comment (arrayslayer) [inline data/projects.js]: Done

Comment (arrayslayer) [inline data/projects.js]: screenshot from 2017-06-27 16-22-39
Here's the screenshot you requested earlier with the developer's involved field modified for a project.

Comment (hemangsk) [inline data/projects.js]: That is awesome 🎉

**Comment by hemangsk** [comment]
@arrayslayer This is all set for merge,  right now the PR has 4 commits
https://github.com/coala/projects/pull/312/commits

- You can drop the ```	
Added Developers Involved Field for Projects In Progress``` commit as your ```projects.js: Add developers_involved field``` already does that the right way.

- Squash the ```projects.js: Rectify syntax error``` commit into ```projects.js: Add developers_involved field``` commit because the former one is a fix up commit right., it needn't be done in a separate commit.

**Comment by arrayslayer** [comment]
I did the rebase but then it doesn't work anymore. I did a separate git clone of the original project repo and even that doesn't work appropriately. Is the current master branch broken or am I doing something wrong?

**Comment by hemangsk** [comment]
Hey,  rebase is successful, just drop the redundant commit (b41b32b)and squash these two (b8a460a and 183f4fa)

**Comment by hemangsk** [comment]
ack 682a17e

**Pull request merged**